TSC-Charter.md

```diff
@@ -36,18 +36,19 @@ approved by the Board.
 
 ## Section 4. Establishment of the TSC.  
 
-TSC memberships are not time-limited. There is no fixed size of the TSC.
-However, the expected target is between 6 and 12, to ensure adequate
-coverage of important areas of expertise, balanced with the ability to
-make decisions efficiently.
+TSC memberships are not time-limited. There is no maximum size of the TSC.
+The size is expected to vary in order to ensure adequate coverage of important
+areas of expertise, balanced with the ability to make decisions efficiently.
+The TSC must have at least three members.
```

Comment:
Perhaps add: "Members are expected to speak and act in the best interests of the Node.js Foundation and open source projects."